src/bindings/python/tests/test_inference_engine/test_compiled_model.py

```diff
@@ -29,8 +29,8 @@ def test_get_property(device):
         pytest.skip("Can't run on ARM plugin due-to CPU dependent test")
     func = core.read_model(model=test_net_xml, weights=test_net_bin)
     exec_net = core.compile_model(func, device)
-    config = exec_net.get_property("PERF_COUNT")
-    assert config == "NO"
+    profiling_enabled = exec_net.get_property("PERF_COUNT")
```

Comment:
@nkogteva 
In new C++ configuration API PERF_COUNT is a boolean value
Before this change CPU plugin still returned "YES/NO" even for new API, which was implicitly converted to bool in case of C++ API.
No such conversion is made for Python API.
Now we want to return proper type (boolean) so no conversion is needed.
Let me know if there are some other python code (maybe demos?) which can be affected by this PERF_COUNT change.
@apankratovantonp @ilya-lavrenov FYI

Reply:
@EgorDuplensky At least benchmark_app tool and speech_sample are affected. cc @vladimir-dudnik to comment from demo/samples team

Reply:
Thanks. I guess we can merge this then.